controller/loadBasicPokemonDetails.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+let fs = require('fs');
+let basicPokemonDetail = require('../models/basicPokemonDetail.js');
+let pokemonListPath = require('../resources/json/pokemonlist.json'); 
+
+function loadBasicPokemonDetails() {
+    console.log('Loading Basic Pokemon Details to MongoDB');
+    let data = JSON.stringify(pokemonListPath, function(key,value) {
+        var pokemonID, pokemonName;
+        if(key != 'name' && typeof value.name != 'undefined') {
```

Comment:
PS.: the only thing I'm scare off by using the typeof is if the value instead of being `undefined` is `null`, which would evaluate to `"object"`. JavaScript has some scary things here and there :S